CODEOWNERS

```
@@ -14,9 +14,12 @@ azure-pipelines.yml @garyverhaegen-da
 azure-cron.yml      @garyverhaegen-da
 
 # Language
-/compiler/          @hurryabit
-/libs-haskell/      @hurryabit
-/daml-assistant/    @associahedron
+/compiler/          @hurryabit @cocreature
+/libs-haskell/      @hurryabit @cocreature
+/daml-assistant/    @associahedron @hurryabit @associahedron
```

```suggestion
/daml-assistant/    @associahedron @hurryabit @cocreature 
```

While @associahedron is certainly the ultimate head of DAML assistant, you probably wanted to add me.

---

Moved to https://github.com/digital-asset/daml/pull/4306